src/util.js

```diff
@@ -13,6 +13,12 @@ function resolvePath(p) {
   return path.resolve(process.cwd(), p);
 }
 
+function getFilteredArgs(rawArgs) {
```

Comment:
If this function gets merged (see other question) but it's only used in `e`, probably better to put it there until its needed by other scripts?

Reply:
o yes good point! will consolidate 🙇‍♀